quickstart/dotnet/Redistest/Redistest.csproj

```diff
@@ -52,11 +53,11 @@
     <None Include="App.config" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="StackExchange.Redis">
-      <Version>2.*</Version>
+    <PackageReference Include="Microsoft.Azure.StackExchangeRedis">
+      <Version>3.1.0</Version>
```

Comment:
Shal we use `3.*` to pull in the latest update w/o breaking change?

Reply:
fixed